sdk/core/azure-core/vcpkg.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
This a bit overlaps with the `vcpkg/vcpkg.json`. Is it possible to integrate them together more tightly / reuse the same file?

If not, is it possible to drop `name` and `version` from these files?

Reply:
looks like we can use VCPKG_MANIFEST_DIR option and re-use the same vcpkg.json we already have :)

Reply:
the problem is that our current vcpkg.json is only a template that requires some pre-proccesing before it is useful. We remove the header lines and set the version value during release :)

Reply:
@vhvb1989, vcpkg would ignore fields that start with `$` (https://vcpkg.readthedocs.io/en/latest/specifications/manifests/). So, if they allow manifests with no version, we can rename `"version-semver"` to `"$version-semver"`, and then rename it back during the install step: see `az_vcpkg_portfile_prep` cmake macro - it drops the copyright comment, we'll just do the similar thing with the `"$version-semver"`.

Reply:
or maybe, they won't complain at all, that our `version-semver` looks like `"@AZ_LIBRARY_VERSION@"`, then it's even easier - nothing needs to be done.

Reply:
The next step would be an automatic procedure to sync the files